daq_macros.py

```diff
@@ -3201,7 +3202,7 @@ def zebraVecDaqSetup(angle_start,imgWidth,exposurePeriodPerImage,numImages,fileP
   exposureTimePerImage =  exposurePeriodPerImage - detector_dead_time
   yield from zebra_daq_prep()
```

Comment:
Does this (also others?) need to be changed to zebra_daq_prep(zebra) as well?

Reply:
yes :( looks like I missed at least one... will need to check over things

Reply:
fixed in https://github.com/NSLS-II/lsdc/pull/101/commits/258e1befc4d7365a64d93d84bf193806ed91a7e0